fe/src/main/java/org/apache/doris/system/Backend.java

```diff
@@ -94,6 +94,9 @@
     // this field is set by tablet report, and just for metric monitor, no need to persist.
     private AtomicLong tabletMaxCompactionScore = new AtomicLong(0);
 
+    // additional backendStatus information for BE, display in JSON format
+    private BackendStatus backendStatus;
```

Comment:
```suggestion
    private BackendStatus backendStatus = new BackendStatus();
```
And remove all constructors below.